src/components/families/family.model.js

```diff
@@ -346,9 +346,10 @@ Family.add( 'Permissions', {
 		MARESupportGroupLeader: { type: Types.Boolean, label: 'MARE support group leader', default: false, initial: true },
 		MARESupportGroupParticipant: { type: Types.Boolean, label: 'MARE support group participant', default: false, initial: true },
 		familyStoryCreated: { type: Types.Boolean, label: 'family story created', default: false, initial: true },
-		videoCreated: { type: Types.Boolean, label: 'video created', default: false, initial: true }
-	},
+		videoCreated: { type: Types.Boolean, label: 'video created', default: false, initial: true },
 		visitingResource: { type: Types.Boolean, label: 'Visiting Resource', default: false, initial: true },
```

Comment:
The `familyServices` section looks good now, thanks for making that change.  One last small formatting nit, please remove the trailing comma after the `visitingResource` field and we're good to go!